README.md

```diff
@@ -111,7 +111,9 @@ To run the bot in simulation mode, try this command:
 
 _Note for Windows Users: You should use a [Bash Shell][bash] to follow the steps below. This will give you a UNIX environment in which to run your commands and will enable the `./scripts/build.sh` bash script to work correctly._
 
-To compile Kelp from source:
+_Note for MacOS Users: Running [install-macos.sh][install-macos-script] should automate the steps below. (including cloning the Kelp repo to the correct Go folder) Remember, manual installation of PostgreSQL and Docker are additionally required._
```

Comment:
just noticed that your script does cloning of the repo too -- I remember we had discussed this last time and how we can take advantage of that. that's super cool.

Can you add to your note here that for MacOS you don't need to clone the directory but can download the automated install script from here and that it will also clone the GitHub repo in the correct location -- and link to the script they should run.